docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  front:
+    build: .
+    tty: true
+    ports:
+      - "3001:3000"
+      - "4000:4000"
+    volumes:
+      - ./app:/app
+    command: yarn dev
```

```suggestion
      - ./app:/app
      - node_modules:/app/node_modules/
    command: yarn dev
volumes:
  node_modules: 
```

volumeを設定してやると、
`docker-compose up -d --build` で一発で起動できるからvolumeの設定があってもいいかもね。

---

```shell
$ git clone https://github.com/ky-hy/nnecstasy-frontend.git
$ cd nnecstasy-frontend
$ docker-compose up -d --build
```
ってこと？
これでやってみたんやけど、node_modulesがホスト側に入らんな。。。

---

ホスト側に入れる意味ってあるんやっけ？

---

なくてもいい気がするが。。

---

このままでいい気がするので、このままで行きましょう、